awesomemotive/file-upload-types
# Update sass dependencies

Fixes #58 

It is required to move forward with dependabot PR requests #56 and #57

Testing procedure:
1. Clone plugin directory
2. Run `npm install` inside plugin directory
3. Run `gulp css` and `gulp watch` and expect no errors
